advanced_functionality/working_with_tfrecords/tfrecord-transformer-container/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2.7
```

Comment:
is there a special reason to use 2.7? generally new code should be python 3.6

Reply:
No special reason, updated to 3.6